packages/contracts/contracts/ERC2981ContractWideRoyalties.sol

```diff
@@ -0,0 +1,32 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.4;
+
+import '@openzeppelin/contracts/utils/introspection/ERC165.sol';
+
+import './ERC2981Base.sol';
+
+/// @dev This is a contract used to add ERC2981 support to ERC721 and 1155
+/// @dev This implementation has the same royalties for each and every tokens
+abstract contract ERC2981ContractWideRoyalties is ERC2981Base {
+    RoyaltyInfo private _royalties;
+
+    /// @dev Sets token royalties
+    /// @param recipient recipient of the royalties
+    /// @param value percentage (using 2 decimals - 10000 = 100, 0 = 0)
+    function _setRoyalties(address recipient, uint256 value) internal {
+        require(value <= 10000, 'ERC2981Royalties: Too high');
+        _royalties = RoyaltyInfo(recipient, uint24(value));
+    }
+
+    /// @inheritdoc	IERC2981Royalties
+    function royaltyInfo(uint256, uint256 value)
+        external
+        view
+        override
+        returns (address receiver, uint256 royaltyAmount)
+    {
+        RoyaltyInfo memory royalties = _royalties;
+        receiver = royalties.recipient;
+        royaltyAmount = (value * royalties.amount) / 10000;
```

Comment:
This looks like it's missing a return value?

Reply:
receiver and royaltyAmount are defined in `returns` in the function signature. This way also works.

Reply:
receiver and royaltyAmount are defined in returns in the function signature. This way also works.

Reply:
that's cool